mobify/platform-scaffold
# AMP: fix header bar

**JIRA**: n/a

## Changes
- remove Search button in header till we support Search on AMP
- Add links to Location and Add to Cart page. 
- Fix header bar styles 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](http://localhost:3000/potions/eye-of-newt.html)
- Check header to make sure that everything in header is working: menu: open amp sidebar, store: go MP store page, cart: go to PWA cart page.
